contrib/cirrus/lib.sh

```diff
@@ -312,9 +312,9 @@ install_conmon(){
         GOPATH $GOPATH
         CRIO_COMMIT $CRIO_COMMIT
     "
-    DEST="$GOPATH/src/github.com/kubernetes-sigs/cri-o.git"
+    DEST="$GOPATH/src/github.com/cri-o/cri-o.git"
     rm -rf "$DEST"
-    ooe.sh git clone https://github.com/kubernetes-sigs/cri-o.git "$DEST"
+    ooe.sh git clone https://github.com/cri-o/cri-o.git "$DEST"
```

Comment:
Testing testing testing

Reply:
Alright, I can add comments here, but not on other files in the diff.
What the hell, Github.